src/main/java/rest/RateResource.kt

```diff
@@ -12,6 +12,7 @@ import javax.ws.rs.core.MediaType
 import javax.ws.rs.core.Response
 
 @Path("rates")
+@Stateless
```

Comment:
Why is this now stateless? Was it tested?

Reply:
Fixed